utils.js

```diff
@@ -2878,40 +2878,37 @@ export function parseGradleProperties(rawOutput) {
  */
 export function executeParallelGradleProperties(dir, rootPath, allProjectsStr) {
   const defaultProps = {
-    rootProject: subProject,
+    rootProject: "root",
```

Comment:
Instead of hardcoding, can we set this to `spstr` from the for loop?

Reply:
I'll have look, this was just a quick fix/hack to get things working.